variables.tf

```
@@ -300,6 +300,8 @@ variable "regions" {
   type = object({
     additional_allowed_service_actions_per_region    = optional(map(list(string)), {})
     allowed_regions                                  = list(string)
+    enable_edge_service_actions                      = optional(bool, false)
+    enable_cdk_service_actions                       = optional(bool, false)
```

```suggestion
    enable_cdk_actions                               = optional(bool, false)
    enable_lambda_at_edge_actions                    = optional(bool, false)
```

Updated to call it Lambda@Edge and removed "service" to keep the field names short - what do you think?

---

Aligned and decided to keep it like it is